README.md

```diff
@@ -112,9 +129,69 @@ function MyComponent() {
 }
 ```
 
+### Icons
+
+All our icons are displayed in a gallery in [Storybook](https://space-kit.netlify.com/?path=/story/space-kit--icons).
+
+Note that there are no styles or classes applied to the SVGs by default; you'll have to add a `width` and `height` to see the icons; and apply a text color to color them.
+
+All our icons are SVG files stored in [`./icons/src/svgs`](./icons/src/svgs). There are scripts set up to convert these SVGs into React components, and then to transpile those files for consumption. These conversions and transpilations are `.gitignore`'ed, so they are not mantained in source control.
+
+These icons are _not_ open source and are only licensed for use in this project. See [license](./icons/LICENSE.md) for more details.
+
+Please see [#developing-space-kit-icons](#icons-1) for instructions on adding new icons.
+
+#### Example
+
+```js
+import React from "react";
+import { IconServices } from "@apollo/space-kit/icons/IconServices";
+
+export const IconServiceItem: React.FC = () => (
+  <div className="w-5 h-5">
+    <IconServices className="w-full h-full text-teal" />
+  </div>
+);
+```
+
+#### FAQ
+
+##### My icons aren't showing up in the UI
+
+Make sure that the icon component has a width and height applied to it. That can mean applying classes or styles directly to the Icon component, or setting the component to have `height: 100%` and `width: 100%` and then applying a size to the containing element.
+
+##### Why can't I import from `IconServices` from `@apollo/space-kit/icons`?
+
+My goal was to minimze the bundle size increase caused by using these icons. If I had named exports from `space-kit/icons`, then the user would have to make sure they are tree-shaking to only import the icons they are actually using. `engine-frontend` is _not_ yet tree-shaking, so we decided to not make the imports an option.
+
+##### Why does each icon have a named export instead of a default export?
+
+The engine-frontend team and Apollo OSS teams have decided to not use default exports; this continues that trend.
+
 ## Developing Space Kit
 
+### Icons
+
+To add new icons, add SVGs to [`./icons/src/svgs`](./icons/src/svgs) and open a pull request. The React components will be generated and the TypeScript will be transpiled automatically after merging to `master`.
```

Comment:
instead:

To add new icons, add SVGs to [`./icons/src/svgs/some_folder_name`](./icons/src/svgs) in a folder name that makes sense. The colors of this icon will not be maintained when the React components are generated. All colors that are black will be converted to be configurable and all other colors will be unchangeable when consumed.